website/blog/2018-05-07-using-typescript-with-react-native.md

```diff
@@ -10,7 +10,17 @@ category: engineering
 
 JavaScript! We all love it. But some of us also love [types](https://en.wikipedia.org/wiki/Data_type). Luckily, options exist to add stronger types to JavaScript. My favourite is [TypeScript](https://www.typescriptlang.org), but React Native supports [Flow](https://flow.org) out of the box. Which you prefer is a matter of preference, they each have their own approach on how to add the magic of types to JavaScript. Today, we're going to look at how to use TypeScript in React Native apps.
 
-We'll be using Microsoft's [TypeScript-React-Native-Starter](https://github.com/Microsoft/TypeScript-React-Native-Starter) repo as a guide.
+This post uses Microsoft's [TypeScript-React-Native-Starter](https://github.com/Microsoft/TypeScript-React-Native-Starter) repo as a guide.
+
+**Update**: Since this blog post was written, [Babel 7 was released with TypeScript support built-in](https://blogs.msdn.microsoft.com/typescript/2018/08/27/typescript-and-babel-7/). Babel 7 replaces all the set up described in this blog post with just one command:
```

Comment:
I think "built-in" might be a bit misleading since it requires [`preset-typescript`](https://babeljs.io/docs/en/babel-preset-typescript) to be configured before it will work, similar to other presets / plugins. (`preset-typescript` was developed in parallel with Babel 7 and took advantage of new functionality that was not available in 6, so its less a feature of Babel 7 than a product of its capabilities.)

Reply:
Ah, yeah, “integrated” might be more accurate.